tools/obscuroscan_v3/frontend/components/modules/batches/data.tsx

```diff
@@ -0,0 +1,39 @@
+import {
```

Comment:
These seem to be constats yeah ? If so can the file to be renamed to something as such ? It's a small nit but data is super generic, would be great to have it locked down more imo.